### PR TITLE
Delete unused code from Registry

### PR DESCRIPTION
**Pull request by myyra**

### CODE REVIEW

File: Core/Barcelona/Wrappers/Registry.swift [hunk 1]
```diff
@@ -50,24 +46,8 @@ class Registry {
...
-    func imHandle(withID id: String, onService service: IMService) -> IMHandle? {
-        guard let account = bestAccount(for: service) else {
-            return nil
-        }
-
-        return imHandle(withID: id, onAccount: account)
-    }
-
     func imHandle(withID id: String, onAccount account: IMAccount) -> IMHandle? {
-        account.imHandle(withID: id) ?? account.existingIMHandle(withID: id)
```
Comment (itsjunetime) [inline Core/Barcelona/Wrappers/Registry.swift:70]: Why did you remove the nil-coalescing stuff?

Comment (myyra) [reply Core/Barcelona/Wrappers/Registry.swift:70]: According to the compiler it doesn't do anything.

Comment (myyra) [reply Core/Barcelona/Wrappers/Registry.swift:70]: 20230406Ea1Skr8z

Comment (myyra) [reply Core/Barcelona/Wrappers/Registry.swift:70]: > Left side of nil coalescing operator '??' has non-optional type 'IMHandle', so the right side is never used
